src/digraphs.c

```diff
@@ -701,7 +701,7 @@ static Obj FuncIS_ANTISYMMETRIC_DIGRAPH(Obj self, Obj adj) {
           level++;
           nbs = ELM_PLIST(adj, j);
           stack += 4;
-          stack[0] = INT_INTOBJ(ADDR_OBJ(nbs)[k]);
+          stack[0] = INT_INTOBJ(ELM_LIST(nbs, k));
```

Comment:
It's not that `ADDR_OBJ(nbs)` isn't valid, it's that when `nbs` is a range, you cannot get the `k`th element of the range by `ADDR_OBJ(nbs)[k]`.